### DISCUSSION TITLE
Spacecharge effect

### DISCUSSION THREAD

**Discussion by xh-ui** [epochpic/epoch#671] [Q&A]
Dear all,

I'm wondering if the spacecharge effect is considered in EPOCH, since the manual doesn't refer to it.

**Comment by Status-Mirror** [comment]
Hi @xh-ui,

I'm finding the phrase "spacecharge effect" ambiguous. Fields in EPOCH are updated according to Maxwell's equations, so if a charge-separation occurs over the simulation run-time, you would be able to model the space-charge.

However, EPOCH always assumes each cell is neutral on simulation start-up. If you initialise a simulation with a charge separation (e.g. a static ion population on one side, and a static electron population on the other), EPOCH will not initialise the physical fields between these two species. If these are problems you'd like to model, you can initialise the fields yourself using the [field block](https://epochpic.github.io/documentation/input_deck/input_deck_fields.html).

Hope this helps,
Stuart

**Reply by xh-ui** [reply]
Thank you ! I think I understand you. I missed the basement of EPOCH with using Maxwell's equations before.
